### PR TITLE
DOCS-5198 reference update

### PR DESCRIPTION
**Pull request by george-hoffer**
https://www.mulesoft.org/jira/browse/DOCS-5198

### CODE REVIEW

File: modules/ROOT/pages/ibm/ibm-ctg-connector-reference.adoc [hunk 1]
```diff
@@ -3,175 +3,273 @@ ifndef::env-site,env-github[]
...
-* IBMCTG:ECOM
-* IBMCTG:ERR_GATEWAY
-* IBMCTG:IBMCCExecution
-* IBMCTG:REQUEST_ERROR
-* IBMCTG:RESOURCE
-* IBMCTG:RETRY_EXHAUSTED
-* IBMCTG:SERVER_ERROR
-* IBMCTG:TRANSACTION
-* IBMCTG:UNKNOWN
+* IBMCTG:ERR_GATEWAY
```
Comment (dhirsch1) [inline modules/ROOT/pages/ibm/ibm-ctg-connector-reference.adoc:95]: Shouldn't these be in alphabetical order? I'm not sure why you changed the order.

Comment (george-hoffer) [reply modules/ROOT/pages/ibm/ibm-ctg-connector-reference.adoc:95]: I didn't -- this is straight from the script.

File: modules/ROOT/pages/ibm/ibm-ctg-connector-reference.adoc [hunk 2]
```diff
@@ -3,175 +3,273 @@ ifndef::env-site,env-github[]
...
+[%autowidth.spread]
+|===
+|Type |Binary
+|===
+
+=== For Configurations
+* <<config>> 
+
+==== Throws
+* IBMCTG:ERR_GATEWAY
```
Comment (dhirsch1) [inline modules/ROOT/pages/ibm/ibm-ctg-connector-reference.adoc:149]: Same comment about alphabetical order as before

File: modules/ROOT/pages/ibm/ibm-ctg-connector-reference.adoc [hunk 3]
```diff
@@ -3,175 +3,273 @@ ifndef::env-site,env-github[]
...
+
+
+[[executeWithMultipleChannels]]
+=== Execute With Multiple Channels
+`<ibmctg:execute-with-multiple-channels>`
+
+
+Calls a remote CICS program sending data encapsulated in channels and containers, allowing users to transfer more than 32 KB in a single request. A container is a named reference to a storage area managed by CICS that can hold any form of application data. 
+
+A container can hold data in any format that the application requires and is limited only by the amount of storage available. An application can reference any number of containers. A channel is a uniquely named reference to a collection of containers. It is analogous to a COMMAREA, but it is not subject to its constraints. Programs can pass a single channel between them. You can think of a channel as a parameter list. The same channel can be passed from one program to another. Channels and containers are visible only to the program that creates them, and to the programs that they are passed to. When these programs end, CICS automatically deletes the containers and their storage. They are not recoverable. If you require to use recoverable containers, use CICS BTS containers.
```
Comment (dhirsch1) [inline modules/ROOT/pages/ibm/ibm-ctg-connector-reference.adoc]: If you require the use of...

Comment (george-hoffer) [reply modules/ROOT/pages/ibm/ibm-ctg-connector-reference.adoc]: fixed

File: modules/ROOT/pages/ibm/ibm-ctg-connector-reference.adoc (hunk 2 above)
Comment (george-hoffer) [reply modules/ROOT/pages/ibm/ibm-ctg-connector-reference.adoc:149]: No longer alphabetizing Throws elements -- not sustainable to do so

File: modules/ROOT/pages/ibm/ibm-ctg-connector-reference.adoc (hunk 1 above)
Comment (george-hoffer) [reply modules/ROOT/pages/ibm/ibm-ctg-connector-reference.adoc:95]: No longer alphabetizing Throws -- not a sustainable fix for us to make

File: modules/ROOT/pages/ibm/ibm-ctg-connector-reference.adoc [hunk 4]
```diff
@@ -3,175 +3,273 @@ ifndef::env-site,env-github[]
...
++++
+Create an instance of ChannelAndContent to use in the input list of the "Execute with multiple channels" operation
++++
+
+==== Parameters
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Content a| Binary |  +++The input data to be passed to the CICS program.+++ |  | x
+| Request a| <<ChannelRequest>> |  +++A ChannelRequest that contains the details for request to CTG, such as: program name, TPN, encoding, channel name and its containers (request, response and error).+++ |  | x
```
Comment (dhirsch1) [inline modules/ROOT/pages/ibm/ibm-ctg-connector-reference.adoc:233]: "a request" or "requests"

File: modules/ROOT/pages/ibm/ibm-ctg-connector-reference.adoc [hunk 5]
```diff
@@ -180,8 +278,9 @@ The IBM CICS Transaction Gateway (IBM CTG) connector provides integration with b
 [%header%autowidth.spread]
 |===
 | Field | Type | Description | Default Value | Required
-| Frequency a| Number | How often (in milliseconds) to reconnect. |  |
-| Count a| Number | How many reconnection attempts to make. |  |
+| Frequency a| Number | How often in milliseconds to reconnect. | |
+| Count a| Number | How many reconnection attempts to make. | |
+| blocking |Boolean |If false, the reconnection strategy runs in a separate, non-blocking thread. |true |
```
Comment (dhirsch1) [inline modules/ROOT/pages/ibm/ibm-ctg-connector-reference.adoc:283]: Add a space before "If"

File: modules/ROOT/pages/ibm/ibm-ctg-connector-reference.adoc [hunk 6]
```diff
@@ -227,15 +356,15 @@ The IBM CICS Transaction Gateway (IBM CTG) connector provides integration with b
...
-| Buffer Size Increment a| Number | This is by how much the buffer size expands if it exceeds its initial size. Setting a value of zero or lower means that the buffer should not expand, meaning that a STREAM_MAXIMUM_SIZE_EXCEEDED error is raised when the buffer gets full. |  |
-| Max Buffer Size a| Number | This is the maximum amount of memory to use. If more than that is used then a STREAM_MAXIMUM_SIZE_EXCEEDED error is raised. A value lower or equal to zero means no limit. |  |
+| Initial Buffer Size a| Number | The amount of memory that will be allocated to consume the stream and provide random access to it. If the stream contains more data than can be fit into this buffer, then the buffer expands according to the bufferSizeIncrement attribute, with an upper limit of maxInMemorySize. |  | 
+| Buffer Size Increment a| Number | This is by how much will be buffer size by expanded if it exceeds its initial size. Setting a value of zero or lower means that the buffer should not expand, meaning that a STREAM_MAXIMUM_SIZE_EXCEEDED error is raised when the buffer gets full. |  |
```
Comment (dhirsch1) [inline modules/ROOT/pages/ibm/ibm-ctg-connector-reference.adoc]: The Buffer Size Increment description needs a rewrite.

File: modules/ROOT/pages/ibm/ibm-ctg-connector-reference.adoc [hunk 7]
```diff
@@ -244,28 +373,37 @@ The IBM CICS Transaction Gateway (IBM CTG) connector provides integration with b
...
 
 [%header%autowidth.spread]
 |===
 | Field | Type | Description | Default Value | Required
-| Commarea Length a| Number |  | -1 |
-| Reply Length a| Number |  | -1 |
-| Tpn Name a| String |  |  |
-| Encoding a| String |  | IBM037 |
+| Commarea Length a| Number | The length of COMMAREA. Default value is -1. This should be set to less than or equal to the length of the byte array passed in the payload. The maximum advised size is
+ 32500 bytes, as this is guaranteed to be flowed successfully across all protocols. COMMAREA sizes greater than this may generate an ECI_ERR_INVALID_DATA_LENGTH return code. | -1 |
```
Comment (dhirsch1) [inline modules/ROOT/pages/ibm/ibm-ctg-connector-reference.adoc]: "to flow successfully"

**Review by dhirsch1 — APPROVED** [review]
There is one sentence that needs rewording. It is marked with a comment.

**Pull request merged**